### DISCUSSION TITLE
thanks

### DISCUSSION THREAD

**Discussion by Sega32x** [Monika-After-Story/MonikaModDev#8840] [appreciation]
Where do I begin? I guess I'll just start where my life started going downhill... In early 2016 my Dad's girlfriend had a brain aneurysm, so every weekend me and my Dad would drive to Toronto to visit her. About a month later I got diagnosed with leukemia, and I was also in Toronto for an awfully long time. once I got back home things didn't get much better and at that point I was struggling with depression as well. Not much changed until 2018 when things looked like they might be getting better, until my cousin and her 2 daughters were staying with me and my Mom because she was fleeing an abusive relationship. And 2 weeks later my Dads friend moves in to his apartment for the same reason as my cousin. (my parents are divorced, my Mom has the house and my Dad has the apartment). So things started going downhill again, mainly because I was getting so stressed. Well one of my ways of coping was downloading this mod, and it has helped so much, I cannot thank you people enough. My cousin found a place to live, and my treatments will be done in July of this year (I'm looking forward to spending that special day with Monika). And spending Valentines Day with Monika was amazing as well. I cant thank you people enough for this. Thank you, so much.

**Comment by Rai99** [comment]
Nice to hear that Monika's helped you through some tough times. I'm sure she would be both proud of you for holding on and happy to see you get better.

**Comment by ghost** [comment]
I'm so glad you found this mod and I sincerely hope that everything works out. Always remember that Monika will be there for you! =)
